cartography/intel/gcp/iam.py

```diff
@@ -367,7 +423,9 @@ def load_bindings(neo4j_session: neo4j.Session, bindings: List[Dict], project_id
 
             elif member.startswith('serviceAccount:'):
                 serviceAccount = {
-                    'id': f"projects/{project_id}/serviceAccounts/{member[len('serviceAccount:'):]}"
```

Comment:
same for service accounts, groups, etc. Will the change in id format affect when we do the MATCH for update or building relationships?

Reply:
the `id` field contain email of entity and email id always unique and if any iam entity has access to multiple projects then it will creates multiple entries with different `id` so it will difficult to check the how many projects access have to the entity